lib/provisioning/Provisioner.js

```diff
@@ -12,9 +12,26 @@ function Provisioner(ircBridge) {
     let as = this._ircBridge.getAppServiceBridge().appService;
     let self = this;
 
+    if (!(as.app.use && as.app.get && as.app.post)) {
+        log.warn('Could not start provisioning API');
+        return;
```

Comment:
I wonder if we should be continuing if this is the case. I guess it depends on whether the admin wants to run the provisioning API. If they do (is this in the config?), then we should definitely hard-fail. If it isn't, then failing like this feels fine to me.